keras/models.py

```diff
@@ -321,6 +325,7 @@ def __init__(self):
         self.params = [] # learnable
         self.regularizers = [] # same size as params
         self.constraints = [] # same size as params
+        self.cost_updates = [] # NOT the same size as params
```

Comment:
Yes, it's a presentation/comments issue. The comment "same size as params" means there is a 1:1 mapping between the arrays. The comment "NOT the same size as params" is not informative as to what the array contains.